### PR TITLE
Weekly pagination of Login/Logoff table 

### PR DESCRIPTION
**Pull request by ssahara**
This request makes the pagination of User Logins/Logouts table weekly, from Monday to Sunday. 

It may be convenient for admin to allow checking user login events during working days of every weeks. This request also covers pull request #8 by @leplastic about 1 year ago.

### CODE REVIEW

File: admin.php [hunk 1]
```diff
@@ -104,14 +104,14 @@ function html() {
         echo '</table>';
 
         echo '<div class="pagenav">';
-        if($max < time()-(7*24*60*60)){
+        if ($max <= $_SERVER['REQUEST_TIME']){
```
Comment (splitbrain) [inline admin.php]: you should not rely on this HTTP related header. I guess replacing it by time() should work just as well.

Comment (ssahara) [inline admin.php]: Thanks for your comment. I have now replaced `$_SEVER['REQUEST\TIME']` to `time()` call.

**Comment by ssahara** [comment]
Hi @splitbrain ,
I had rebased my branch in order to update my some DW sites, and I also found that this PR automatically follows changes! As a result, this PR does not look tidy. If you like any changes, I will create new PR separately.

**Comment by splitbrain** [comment]
yes, please open a new pr

**Pull request closed without merging**